lib/server_env.rb

```diff
@@ -4,4 +4,8 @@ module ServerEnv
   def self.instance_index
     CloudFoundry.instance_index || 0
   end
+
+  def self.first_instance?
+    self.instance_index == 0
```

Comment:
I would probably call this just as `instance_index` rather than using `self` but that's just me -- what do you think?